pkg/sync/sync.go

```diff
@@ -441,7 +441,7 @@ func (t *Task) checkAndExpandReplica(address string, size int64) error {
 		return err
 	}
 	if replicaSize > size {
-		return fmt.Errorf("cannot add a larger replica to the engine")
+		return fmt.Errorf("cannot create new replica less than size %v", replicaSize)
```

Comment:
This doesn't look right. ReplicaSize is the new replica which going to be added. @shuo-wu can you confirm?

Reply:
You are right. `replicaSize ` is the size of the new/rebuilding replica. `size` is the size of the engine & existing replicas.

Reply:
Thanks for clarifying @yasker and @shuo-wu . I've updated the issue to be an enhancement of the error message instead.